library/build.gradle

```diff
@@ -23,11 +23,16 @@ android {
 
     buildFeatures {
         viewBinding = true
+        buildConfig = false
```

Comment:
This is in theory a breaking change but I think we can make it slide. I don't suppose anyone uses `BuildConfig`.

Reply:
Good catch. I would like to play safe and be sure that we don't introduce anything breaking for 3.x users, so reverted this change.